content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -107,6 +107,22 @@ jobs:
               params:
                 alert_type: failed
                 text: "Failed on get webpack-json : ((pipeline_name))/((site-name))"
+        on_error:
```

Comment:
@mbertrand So as I was finishing this up, I got to thinking, should we add calls to 'ocw-studio-webhook' to the `on_error` and `on_abort` clauses as well? I can't think of a reason _not_ to but I'm not familiar with the app side of equation.

Reply:
Sure, that would be great!  The `status` sent in the webhook can be `errored` for `on_error` and `aborted` for `on_abort`.

Reply:
Okay I will get started!

Reply:
Should I switch the `status` in the `on_failure` webhook calls to `failed` or should it remain as `errored`?

Reply:
Leave it as `errored`.  There is no constant for `failed` in the app currently, and I'm not sure if making a distinction between `errored` and `failed` would be useful.

https://github.com/mitodl/ocw-studio/blob/c8da66e64fe2e2c4b449bdf99f66d5c4a9459da8/websites/constants.py#L92-L110